src/main/riddl/Gateway/actorDefinitions.riddl

```diff
@@ -1,3 +1,3 @@
 actor Member is "The person that is being enrolled"
 actor App is "The applicaiton the user is using to perform this action"
```

Comment:
I would prefer this was specified as an Application definition. I think this was written before the Application/Epic/Actor triangle was established.